pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -62,7 +62,7 @@ func (q *graceTerminateRSList) add(rs *listItem) bool {
 		return false
 	}
 
-	klog.V(5).Infof("Adding rs %v to graceful delete rsList", rs)
+	klog.V(5).InfoS("Adding rs item to graceful delete rsList", "realServer", rs)
```

Comment:
Throughout this code we use "rs" as a shorthand for "realServer". Please remember that from perspective of cluster administrator that doesn't know code it might be unclear what "rs" stands for. Please make sure that all logs expand "rs" to "realServer"

Reply:
Yes, I thought doing this in the first place but wasn't sure if rs is used commonly, I should've reached out to you for this.
Fixed it for every occurrence of rs, thanks for pointing out.